RecoParticleFlow/PFProducer/python/particleFlowBlock_cfi.py

```diff
@@ -209,3 +209,9 @@ def _findIndicesByModule(name):
     particleFlowBlock,
     elementImporters = _addTimingLayer
 )
+
+from Configuration.ProcessModifiers.egamma_lowPt_exclusive_cff import egamma_lowPt_exclusive
+egamma_lowPt_exclusive.toModify(particleFlowBlock.elementImporters[2],
```

Comment:
please use ```_findIndicesByModule``` instead of direct indexing.
A less preferred solution would be to move this change up, to be right after the definition of particleFlowBlock, before all other modifiers are called

Reply:
could you please suggest how exactly should I do it?

Reply:
visually, it looks like  replacing 2 with ```_findIndicesByModule("SuperClusterImporter")[0]``` should work,
perhaps more effective is to do
```python
for imp in particleFlowBlock.elementImporters:
  if imp..importerName.value() == "SuperClusterImporter":
    _scImporter = imp
egamma_lowPt_exclusive.toModify(_scImporter, ...
```